airbyte-integrations/connectors/source-xero/metadata.yaml

```diff
@@ -6,13 +6,18 @@ data:
     oss:
       enabled: true
     cloud:
-      enabled: false
+      enabled: true
   remoteRegistries:
     pypi:
       enabled: true
       packageName: airbyte-source-xero
   releases:
     breakingChanges:
+      3.0.0:
+        upgradeDeadline: "2024-08-30"
+        message:
+          Due to limitations on the number of OAuth users permitted for the Airbyte application, OAuth support has been disabled.
```

Comment:
I don't know the right wording, but want to make this look nicer. /cc @katmarkham